test/Ocelot.AcceptanceTests/Authorization/AuthorizationTests.cs

```diff
@@ -271,6 +271,116 @@ public async Task Should_return_200_OK_with_global_allowed_scopes()
         await ThenTheResponseBodyAsync();
     }
 
+    [Fact]
+    [Trait("Feature", "Space-separated scopes")]
+    public async Task Should_return_200_OK_with_space_separated_scope_single_match()
+    {
+        var port = PortFinder.GetRandomPort();
+        var route = GivenAuthRoute(port);
+        route.AuthenticationOptions.AllowedScopes = ["api", "api.read", "api.write"];
+        var configuration = GivenConfiguration(route);
+        GivenThereIsAConfiguration(configuration);
+
+        await GivenThereIsExternalJwtSigningService(route.AuthenticationOptions.AllowedScopes.ToArray());
+        GivenThereIsAServiceRunningOn(port, HttpStatusCode.OK, "Hello from space-separated test");
```

Comment:
Custom body checking is optional, as is specifying an explicit `HttpStatusCode.OK` status. You can make them implicit by using the response body as the test name:
https://github.com/ThreeMammals/Ocelot/blob/35dbc9d633dc49fe8b5a1f6589ff9e2ec72bf07c/test/Ocelot.AcceptanceTests/Authorization/AuthorizationTests.cs#L265
and
https://github.com/ThreeMammals/Ocelot/blob/35dbc9d633dc49fe8b5a1f6589ff9e2ec72bf07c/test/Ocelot.AcceptanceTests/Authorization/AuthorizationTests.cs#L271